devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy

```diff
@@ -104,6 +110,18 @@ def parseIasMessage(String description) {
 	return getMotionResult(motionActive)
 }
 
+def parseWeLinKMotion(String description) {
```

Comment:
Rather than creating this new function, I'd like to see an update to the existing `parseIasMessage`. Since the eWeLink link device does report when motion stops but doesn't set the Restore Notify bit in the ZoneStatus message you'll need to do something like this:

```
def parseIasMessage(String description) {
	ZoneStatus zs = zigbee.parseZoneStatus(description)
	Boolean motionActive = zs.isAlarm1Set() || zs.isAlarm2Set()
	 if (!(zs.isRestoreReportsSet() || supportsRestoreNotify())) {
		if (motionActive) {
			def timeout = 20
			log.debug "Stopping motion in ${timeout} seconds"
			runIn(timeout, stopMotion)
		}
	}
	return getMotionResult(motionActive)
}
```

def supportsRestoreNotify() {
     getDataValue("manufacturer") == "eWeLink"
}